source/standards/running-services-as-containers.html.md.erb

```diff
@@ -0,0 +1,50 @@
+---
+title: How to run your service as a container
+
+expires: 2018-05-02
+---
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+If your web application is running on [GOV.UK Platform as a Service (PaaS)][] you should follow the [recommended hosting guidance][].
+
+If your web application is running on PaaS you should use:
+
+* [Docker][] to [build and deploy images][], as it’s widely used across GDS and industry
+* [Dockerhub][] or [Amazon EC2 Container Registry][] to host your images following the [PaaS responsibility model][]
+* the recommended [monitoring and logging solution][]
+
+You can also run your service using bare metal Docker on an [Amazon Elastic Compute Cloud (Amazon EC2)][] instance if:
+
+* service uptime is not a priority
+* zero downtime deployments are not required
+* it can be managed by configuration management
+
+You should not run:
+
+* local images on the PaaS, as [deploying bare metal containers][] are not supported
+* Docker containers with root access using the Docker daemon as this has [security implications][]
+
+If your service requires high-availability, you should use the [Amazon Elastic Container Service (Amazon ECS)][] as your container orchestration service. By using [Amazon ECS clustering][] you can protect against unhealthy instances, and balance service workload across corresponding healthy instances.
```

Comment:
do we recommend this for all services on the PaaS? are there any services that don't actually require this and what would be an example?

Reply:
you can only deploy docker images from a registry, the cf push command simply does not support it. Its less a question of should not, the cli simply does not support deployment of a local docker image.